### PR TITLE
TIP-842: Move attribute normalizer to Akeneo\Pim\Structure

### PR DESCRIPTION
**Pull request by arnolanglade**
**Description (for Contributor and Core Developer)**

We are working on the namespace `Akeneo\Pim\Structure`. We moved normalizers related to the following entities in this PR: 

* Attribute
* AttributeOption
* Family
* VariantFamily
* AttributeGroup

Some normalizers have been improved to avoid to extend them in EE.

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | -
| Added legacy Behats               | -
| Added acceptance tests            | -
| Added integration tests           | -
| Changelog updated                 | yes
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | -
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: src/Akeneo/Pim/Structure/Component/Normalizer/Indexing/FamilyNormalizer.php [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?php
 
-namespace Pim\Component\Catalog\Normalizer\Indexing;
+namespace Akeneo\Pim\Structure\Component\Normalizer\Indexing;
```
Comment (ahocquard) [inline src/Akeneo/Pim/Structure/Component/Normalizer/Indexing/FamilyNormalizer.php]: "indexing" directory is for ES right?

File: src/Akeneo/Pim/Structure/Bundle/Resources/config/normalizers.yml [hunk 2]
```diff
@@ -0,0 +1,167 @@
+services:
+    ### Attribute
+    pim_catalog.normalizer.standard.attribute:
```
Comment (ahocquard) [inline src/Akeneo/Pim/Structure/Bundle/Resources/config/normalizers.yml]: you don't change name of the services?

Comment (pierallard) [reply src/Akeneo/Pim/Structure/Bundle/Resources/config/normalizers.yml]: I can answer (cause I already asked this question): Not for now.

File: src/Akeneo/Pim/Structure/Component/Normalizer/Indexing/FamilyNormalizer.php (hunk 1 above)
Comment (ahocquard) [reply src/Akeneo/Pim/Structure/Component/Normalizer/Indexing/FamilyNormalizer.php]: Thanks for your response @arnolanglade  :trollface:

**Pull request merged**